### PR TITLE
Fhir 41378

### PR DESCRIPTION
**Pull request by bvdh**

### CODE REVIEW

**Review by pgalinauskas — APPROVED** [review]

File: input/fsh/FHIRcastUserHibernateContext.fsh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* parameter 1..* MS
+* parameter ^slicing.discriminator.type = #value
+* parameter ^slicing.discriminator.path = "name"
+* parameter ^slicing.rules = #open
+* parameter ^slicing.ordered = false   // can be omitted, since false is the default
+* parameter ^slicing.description = "Reason for logout."
+* parameter contains code 1..1 
+* parameter[code].name = "code"
+* parameter[code] ^short = "Coded reason."
+* parameter[code] ^definition = "The reason the Subscriber sends a userHybernate event."
```
Comment (EricOnFHIR) [inline input/fsh/FHIRcastUserHibernateContext.fsh]: userHibernate not userHybernate

**Review by EricOnFHIR — APPROVED** [review]
I made a few grammatical changes and submitted those to the branch.  With those the PR looks super.

File: input/pagecontent/3-2-4-UserHibernate.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 ### Workflow
 
-User temporarily suspended their session. The user's session will eventually resume.
+User temporarily suspended their session due to a session time-out or other reason. The user's session will eventually resume.
  
-Unlike most of FHIRcast events, `UserHibernate.html` is a statically named event and therefore does not follow the `FHIR-resource`-`[open|close]` syntax.
+Unlike most of FHIRcast events, `UserHibernate` is a statically named event and therefore does not follow the `FHIR-resource`-`[open|close]` syntax.
 
 ### Context
```
Comment (isaacvetter) [inline input/pagecontent/3-2-4-UserHibernate.md:11]: Need to add typical table of contextual keys

File: input/fsh/FHIRcastDiagnosticReportClose.fsh [hunk 3]
```diff
@@ -23,7 +23,7 @@ A logical id of the resource must be provided. The provided `id` SHALL be the sa
 * identifier ^short = "At least one identifier of the DiagnosticReport SHOULD be provided in an [FHIR resource]-close request."
 * identifier ^definition = 
 """
-At least one `identifier` of the DiagnosticReport SHOULD be provided in a [FHIR resource]-close request. This could be a business identifier provided in the [DiagnosticReport-open](3-6-3-DiagnosticReport-open.html) event or a business identifier provided subsequently in a [DiagnosticReport-update](3-6-3-DiagnosticReport-update.html) event.
+At least one `identifier` of the DiagnosticReport SHOULD be provided in a [FHIR resource]-close request. This could be a business identifier provided in the [DiagnosticReport-open](3-6-1-DiagnosticReport-open.html) event or a business identifier provided subsequently in a [DiagnosticReport-update](3-6-3-DiagnosticReport-update.html) event.
```
Comment (isaacvetter) [inline input/fsh/FHIRcastDiagnosticReportClose.fsh:26]: this is an unrelated, but nonsubstantive change.

**Pull request merged**